conformance/utils/suite/features.go

```diff
@@ -189,9 +189,10 @@ var MeshCoreFeatures = sets.New(
 //
 // NOTE: as new feature sets are added they should be inserted into this set.
 var AllFeatures = sets.New[SupportedFeature]().
-	Insert(StandardExtendedFeatures.UnsortedList()...).
-	Insert(ExperimentalExtendedFeatures.UnsortedList()...).
+	Insert(GatewayExtendedFeatures.UnsortedList()...).
+	Insert(ReferenceGrantCoreFeatures.UnsortedList()...).
```

Comment:
shouldnt the same logic also be applied to `HTTPCoreFeatures` , `HTTPExtendedFeatures` & `TLSCoreFeatures` ?

Reply:
I'm not sure I understand the request here? Each of those is added to the set in the three lines below.

Reply:
I think I understand it now - why I'm actually trying to do is have per-resource lists, so `GatewayCoreFeatures` would mean the core features included in just the `Gateway` resource and `HTTPRouteCoreFeatures` would include the core features that are part of the `HTTPRoute` resource. Does that approach make sense to you @arkodg?

Reply:
yeah, maybe this is not tied to this PR, but what I was trying to get at is `HTTPCore` is not needed in this list since it should be part of `GatewayCore`

Reply:
But `GatewayCore` would only be core features that are part of the Gateway resource. I think we also want to separate the features of HTTPRoute from Gateway or Mesh use cases so implementations can claim support for HTTPRoute without Gateway. Maybe it would be clearer if we called the lists `HTTPRouteCoreFeatures` and `GatewayCoreFeatures`?

Reply:
yes :) until now, I was assuming `Gateway` is tied to `Ingress` features (for naming in this package/file)

Reply:
Good call, updated, thanks for calling this out!